lib/Page.js

```diff
@@ -72,6 +73,17 @@ function Page(pageConfig) {
  * Util Methods
  */
 
+function addContentWrapper(pageData) {
+  const $ = cheerio.load(pageData);
+  if ($.root().hasClass(CONTENT_WRAPPER_CLASS)) {
+    return pageData;
+  }
+  $(CONTENT_WRAPPER_CLASS).remove();
```

Comment:
What is the purpose of this line of code? Users should be able to add multiple `container-fluid` if he wants inside the website. They cannot be nested, but they can be used multiple times.

Reply:
@yamgent 

If container-fluid class can be added multiple times. We shouldn't be using `container-fluid` as our page wrapper? It should be unique as using media queries for `container-fluid` affects the user using it within.

We can have our unique page wrapper inherit the responsive values from `container-fluid`, which is `overflow-x: hidden` and `width: 100%`.

Reply:
https://stackoverflow.com/questions/23062439/am-i-only-supposed-to-have-one-bootstrap-3-container

While we are using Bootstrap 4, I don't think the philosophy behind `container-fluid` has changed. So yes, one can use `container-fluid` multiple times. It just cannot be nested, that's all.